doc/src/manual/metaprogramming.md

```diff
@@ -864,6 +864,46 @@ However, we don't do this for a good reason: wrapping the `expr` in a new scope
 also slightly changes the meaning of the expression (the scope of any variables in it),
 while we want `@time` to be usable with minimum impact on the wrapped code.
 
+### Macros and dispatch
+
+Macros, just like ordinary functions, are generic. This means they can also have multiple method definitions, thanks to multiple dispatch:
```

Comment:
@Ismael-VC are you okay with this formulation?

Reply:
How about?

> Macros, just like Julia functions, are generic.

I recommend this instead of *ordinary functions*, because all Julia functions are generic, including **generated** functions and **anonymous** functions.

Reply:
As a mater of fact, macros are actually Julia functions ;)